src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -267,6 +267,27 @@ public static File createTempFile(final String name, final String extension) {
     }
 
     /**
+     * Creates a temp list file that will be deleted on exit after tests are complete.
+     * @param tempFilePrefix Prefix of the file.
+     * @param lines lines to write to the file.
+     * @return A list file in the temporary directory starting with tempFilePrefix, which will be deleted after the program exits.
+     */
+    public static File createTempListFile(final String tempFilePrefix, final String... lines) {
```

Comment:
I don't think this method deserves to live in `BaseTest` (it's too specific) -- either move it to the test class that uses it or to a common utility class if multiple classes use it.